### PR TITLE
feat: [CDS-36915]: service entity consumption  in pipeline studio

### PR DESCRIPTION
**Pull request by bhavya-sinha**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by bhavya-sinha** [comment]
retrigger typecheck

File: src/modules/70-pipeline/components/ManifestSelection/ManifestSelection.tsx [hunk 1]
```diff
@@ -61,13 +72,37 @@ export default function ManifestSelection({ isPropagating, deploymentType }: Man
...
+
+  useEffect(() => {
+    if (!isReadonly && readonly) {
+      refetchServiceData()
+    }
+  }, [readonly])
+  /*************************************Service Entity Related code********************************************************/
+
   const listOfManifests = useMemo(() => {
+    if (!isReadonly && readonly) {
```
Comment (AshwinBhatkal-Harness) [inline src/modules/70-pipeline/components/ManifestSelection/ManifestSelection.tsx:91]: What's the difference between readonly and isReadonly?

Comment (bhavya-sinha) [reply src/modules/70-pipeline/components/ManifestSelection/ManifestSelection.tsx:91]: isReadonly is from pipeline context whereas readonly is fromprops which is readonly mode for K8sservicespec when new entity is selected. I will use a better name for that

**Review by vkbansal — APPROVED** [review]

File: src/modules/70-pipeline/components/ManifestSelection/ManifestSelection.tsx (hunk 1 above)
Comment (AshwinBhatkal-Harness) [reply src/modules/70-pipeline/components/ManifestSelection/ManifestSelection.tsx:91]: So, once caching is implemented we will be removing the "readonly" variable right? If yes, then cool. If not, then naming it something specific would be better if someone else is to debug and work on it

**Review by AshwinBhatkal-Harness — APPROVED** [review]

File: src/modules/70-pipeline/components/ManifestSelection/ManifestSelection.tsx (hunk 1 above)
Comment (bhavya-sinha) [reply src/modules/70-pipeline/components/ManifestSelection/ManifestSelection.tsx]: Yes, after caching, we neednot put this condition. But i renamed it for now for clarity

**Comment by bhavya-sinha** [comment]
retrigger jest

**Review by kapilkumawat86 — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Comment by bhavya-sinha** [comment]
retrigger jest

**Review by vkbansal — APPROVED** [review]

**Pull request merged**